processing/src/main/java/org/apache/druid/query/expression/IPv4AddressMatchExprMacro.java

```diff
@@ -106,13 +106,13 @@ public ExprEval eval(final ObjectBinding bindings)
       private boolean isStringMatch(String stringValue)
       {
         IPv4Address iPv4Address = IPv4AddressExprUtils.parse(stringValue);
-        return iPv4Address != null && subnetString.contains(iPv4Address.toAddressString());
+        return iPv4Address != null && subnetString.prefixContains(iPv4Address.toAddressString());
```

Comment:
1. followed this doc : https://github.com/seancfoley/IPAddress/wiki/Code-Examples-2%3A-Subnet-Containment-and-Matching#comparing-just-the-prefix for the diff b/w contains and prefixContains in the lib - is that the right reference? 
2. can also replace a couple of occurrences in `IPv4AddressBenchmark`

Reply:
1: I think yes, but looking closer I redid some things in the latest commit.
2: oops, missed the benchmark, which is what prompted me to look a bit closer